.gitignore

```diff
@@ -9,4 +9,5 @@ htmlcov
 **.DS_Store
 .env
 .pytest_cache/
-storage/compiled/
\ No newline at end of file
+storage/compiled/
```

Comment:
What's the reasoning behind ignoring these?

Reply:
The tests for sass generate a css file.